platform/openide.io/src/org/openide/io/BridgingIOProvider.java

```diff
@@ -92,9 +93,16 @@ public InputOutput getIO(String name, Action[] actions,
     @Override
     public InputOutput getIO(String name, boolean newIO, Action[] actions,
             IOContainer ioContainer) {
+        Object[] lookupContent;
+        if (ioContainer != null) {
+            lookupContent = Arrays.copyOf(actions, actions.length + 1, Object[].class);
+            lookupContent[actions.length] = ioContainer;
+        } else {
+            lookupContent = actions;
+        }
         return new BridgingInputOutput(
                 providerDelegate.getIO(name, newIO,
-                        Lookups.fixed((Object[]) actions, ioContainer)));
```

Comment:
Do I read the change correctly?

Previously the `Lookup` had two elements one of the type `Action[]` and one of the type `IOContainer`. The new `Lookup` is going to have n-elements of type `Action` and optionally one element of type `IOContainer`.